### PR TITLE
Update error message for weightage and marks

### PR DESCRIPTION
**Pull request by rachelkeh**

### CODE REVIEW

File: src/main/java/taa/command/assessment/AddAssessmentCommand.java [hunk 1]
```diff
@@ -58,7 +58,8 @@ public void checkArgument() throws TaaException {
         if (!Util.isStringDouble(maximumMarksString)) {
```
Comment (leyondlee) [inline src/main/java/taa/command/assessment/AddAssessmentCommand.java:58]: Change to `isStringDouble(maximumMarksString, 2)` to only allow 2dp

File: src/main/java/taa/command/assessment/EditAssessmentCommand.java [hunk 2]
```diff
@@ -65,7 +65,8 @@ public void checkArgument() throws TaaException {
             if (!Util.isStringDouble(newMaximumMarksString)) {
```
Comment (leyondlee) [inline src/main/java/taa/command/assessment/EditAssessmentCommand.java:65]: Change to isStringDouble(maximumMarksString, 2) to only allow 2dp

File: src/main/java/taa/command/assessment/AddAssessmentCommand.java (hunk 1 above)
Comment (rachelkeh) [reply src/main/java/taa/command/assessment/AddAssessmentCommand.java]: ok updated

File: src/main/java/taa/command/assessment/EditAssessmentCommand.java (hunk 2 above)
Comment (rachelkeh) [reply src/main/java/taa/command/assessment/EditAssessmentCommand.java]: ok updated

**Pull request merged**